### ISSUE TITLE
QMK Toolbox won't recognize STM32 bootloader, but QMK CLI does

### ISSUE THREAD

**Issue by twangerine** [qmk/qmk_toolbox#362]
I'm trying to flash a Drop Preonic V3. But can't get the bootloader to show up in QMK Toolbox. I dug around a bit and managed to get the bootloader to show in QMK CLI.

## System Information

 - Operating system: macOS 12.3.1 (Macbook Air 2020, M1 Chip)
 - QMK Toolbox version: 0.2.0
   - tried both Open Native M1 and Open in Rosetta, also tried homebrew and .pkg installs.
 
<img width="1430" alt="Screen Shot 2022-05-01 at 10 07 57 AM" src="https://user-images.githubusercontent.com/4623405/166149779-14ccb2cb-2cde-4b8b-aafd-fd541de32c32.png">

**Comment by fauxpark** [comment]
Duplicate of https://github.com/qmk/qmk_toolbox/issues/358

Just waiting on one more PR, then I can release a new version :)
